From b8ee5dd78e7950bdf49e889a3286a3dc15abb828 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 29 Jun 2020 15:54:45 +0000
Subject: [PATCH 105/118] dt-bindings: arm: amlogic: add A95XF3-AIR bindings

Add the board bindings for the CYX A95XF3-AIR STB device.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 52db715359ed..01eb2889242a 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -166,6 +166,8 @@ properties:
           - enum:
               - amediatech,x96-air-2g
               - amediatech,x96-air-4g
+              - cyx,a95xf3-air-2g
+              - cyx,a95xf3-air-4g
               - hardkernel,odroid-c4
               - khadas,vim3l
               - seirobotics,sei610
-- 
2.31.1

